guides/doc-Planning_Guide/topics/Configuration_Management_Support.adoc

```diff
@@ -9,5 +9,4 @@ Supported combinations of major versions of {RHEL} and hardware architectures fo
 |Red Hat Enterprise Linux 8 |x86_64, ppc_64, s390x
```

Comment:
Bleh, the header of this is also outdated since we ship Puppet 6, not Puppet 5.

Reply:
Sure, but I wanted to kill one beast at a time :)